uint64 SUBSYSTEM_TYPE_GYRO = 1
uint64 SUBSYSTEM_TYPE_ACC = 2
uint64 SUBSYSTEM_TYPE_MAG = 4
uint64 SUBSYSTEM_TYPE_ABSPRESSURE = 8
uint64 SUBSYSTEM_TYPE_DIFFPRESSURE = 16
uint64 SUBSYSTEM_TYPE_GPS = 32
uint64 SUBSYSTEM_TYPE_OPTICALFLOW = 64
uint64 SUBSYSTEM_TYPE_CVPOSITION = 128
uint64 SUBSYSTEM_TYPE_LASERPOSITION = 256
uint64 SUBSYSTEM_TYPE_EXTERNALGROUNDTRUTH = 512
uint64 SUBSYSTEM_TYPE_ANGULARRATECONTROL = 1024
uint64 SUBSYSTEM_TYPE_ATTITUDESTABILIZATION = 2048
uint64 SUBSYSTEM_TYPE_YAWPOSITION = 4096
uint64 SUBSYSTEM_TYPE_ALTITUDECONTROL = 16384
uint64 SUBSYSTEM_TYPE_POSITIONCONTROL = 32768
uint64 SUBSYSTEM_TYPE_MOTORCONTROL = 65536
uint64 SUBSYSTEM_TYPE_RANGEFINDER = 131072

bool present
bool enabled
bool ok
uint64 subsystem_type
